src/stream/src/executor/source/state_table_handler.rs

```diff
@@ -178,14 +178,9 @@ impl<S: StateStore> SourceStateTableHandler<S> {
     where
         SS: SplitMetaData,
     {
-        if states.is_empty() {
-            // TODO should be a clear Error Code
-            bail!("states require not null");
```

Comment:
I don't find this check useful?

Reply:
it is ok to remove the check, no need any more